src/cephadm/cephadm

```diff
@@ -3061,31 +2985,52 @@ def command_bootstrap(ctx):
                '\tcaps osd = allow *\n'
                % (mon_key, admin_key, mgr_id, mgr_key))
 
+    admin_keyring = write_tmp('[client.admin]\n'
+                                  '\tkey = ' + admin_key + '\n',
+                                       uid, gid)
+
     # tmp keyring file
-    tmp_bootstrap_keyring = write_tmp(keyring, uid, gid)
+    bootstrap_keyring = write_tmp(keyring, uid, gid)
+    return (mon_key, mgr_key, admin_key,
+            bootstrap_keyring, admin_keyring)
+
 
-    # create initial monmap, tmp monmap file
+def create_initial_monmap(
+    ctx: CephadmContext,
+    uid: int, gid: int,
+    fsid: str,
+    mon_id: str, mon_addr: str
+) -> Any: # type: ignore
```

Comment:
which type error do you try to avoid here?

Reply:
`Any` return making pylance complaining. Can't recall the actual error, but it's annoying.